_posts/2021-07-16-status-update.md

```diff
@@ -0,0 +1,93 @@
+- [`puppetlabs-stdlib#1193`][puppetlabs-stdlib-pr-1193]: "(MAINT) Remove testing logic for Puppet 4.5", thanks to [b4ldr][b4ldr]
+- [`puppetlabs-stdlib#1192`][puppetlabs-stdlib-pr-1192]: "Stdlib::Http::Method: Add new type for http methods", thanks to [b4ldr][b4ldr]
+- [`pdk-templates#442`][pdk-templates-pr-442]: "Set `skip_branch_with_pr` to true by default in appveyor.yml template", thanks to [TraGicCode][TraGicCode]
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-haproxy`][puppetlabs-haproxy] (`6.1.0`)
+- [`puppetlabs-mysql`][puppetlabs-mysql] (`11.1.0`)
+
+  [puppetlabs-haproxy]: https://github.com/puppetlabs/puppetlabs-haproxy
+  [puppetlabs-mysql]: http://github.com/puppetlabs/puppetlabs-mysql
+  [provision-pr-169]: https://github.com/puppetlabs/provision/pull/169
+  [seanmil]: https://github.com/seanmil
+  [provision-pr-168]: https://github.com/puppetlabs/provision/pull/168
+  [puppetlabs-firewall-pr-996]: https://github.com/puppetlabs/puppetlabs-firewall/pull/996
+  [scoiatael]: https://github.com/scoiatael
+  [puppetlabs-haproxy-pr-489]: https://github.com/puppetlabs/puppetlabs-haproxy/pull/489
+  [trefzer]: https://github.com/trefzer
+  [puppetlabs-haproxy-pr-487]: https://github.com/puppetlabs/puppetlabs-haproxy/pull/487
+  [cocker-cc]: https://github.com/cocker-cc
+  [puppetlabs-kubernetes-pr-527]: https://github.com/puppetlabs/puppetlabs-kubernetes/pull/527
+  [danifr]: https://github.com/danifr
+  [puppetlabs-kubernetes-pr-526]: https://github.com/puppetlabs/puppetlabs-kubernetes/pull/526
+  [puppetlabs-kubernetes-pr-511]: https://github.com/puppetlabs/puppetlabs-kubernetes/pull/511
+  [treydock]: https://github.com/treydock
+  [puppetlabs-mysql-pr-1415]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1415
+  [weastur]: https://github.com/weastur
+  [puppetlabs-mysql-pr-1414]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1414
+  [ghoneycutt]: https://github.com/ghoneycutt
+  [puppetlabs-mysql-pr-1409]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1409
+  [puppetlabs-mysql-pr-1408]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1408
+  [puppetlabs-mysql-pr-1406]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1406
+  [alexjfisher]: https://github.com/alexjfisher
+  [puppetlabs-mysql-pr-1405]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1405
+  [vchepkov]: https://github.com/vchepkov
+  [puppetlabs-stdlib-pr-1193]: https://github.com/puppetlabs/puppetlabs-stdlib/pull/1193
+  [b4ldr]: https://github.com/b4ldr
+  [puppetlabs-stdlib-pr-1192]: https://github.com/puppetlabs/puppetlabs-stdlib/pull/1192
+  [pdk-templates-pr-442]: https://github.com/puppetlabs/pdk-templates/pull/442
+  [TraGicCode]: https://github.com/TraGicCode
+
+## Other Work
```

Comment:
```suggestion
## Other Work

## Tomcat's week

This week we focused on Tomcat module, as tomcat7 is EOL, we've made the necessary changes to solve the issues coming with the removal of it. Also, we worked at a feature which allows you to remove webapps and a PR for tomcat7 removal was opened.   
```

Reply:
@daianamezdrea can we get links to the feature and removal PR?